src/Lecture1.hs

```diff
@@ -104,7 +118,15 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: String -> Int 
+strSum str = sumList (convertStrListToInt (words str))
+    where
+        sumList :: [Int] -> Int
+        sumList [] = 0
+        sumList (x:xs) = x + sumList xs
```

Comment:
There's a standard function `sum` that finds a sum of all elements in the list 🙂 
You can use it instead and probably even implement `strSum` as a one-liner